apps/dashboard/src/components/workflow-editor/schema.ts

```diff
@@ -30,6 +30,7 @@ export const workflowSchema = z.object({
         type: z.nativeEnum(StepTypeEnum),
         _id: z.string(),
         stepId: z.string(),
+        slug: z.literal<StepResponseDto['slug']>('_stp_'),
```

Comment:
the `slug` is used in the workflow nodes, we pass it to be able to build the navigation links, see the code in a file above

Reply:
the `StepResponseDto["slug"]` is a template literal which requires this kind of template